src/applications/edu-benefits/opt-out/routes.jsx

```diff
@@ -0,0 +1,12 @@
+import { createRoutesWithSaveInProgress } from '../../../platform/forms/save-in-progress/helpers';
+import formConfig from './config/form';
+import OptOutApp from './containers/OptOutApp.jsx';
+
+const route = {
+  path: '/',
+  component: OptOutApp,
+  indexRoute: { onEnter: (nextState, replace) => replace('/introduction') },
```

Comment:
Since we're not going to have an introduction page, do we know what we need to change with the routes to make this basically function as just a review and submit page.

Reply:
[An issue](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11429) has been created to discuss this.

Reply:
The introduction and review-and-submit routes are now filtered out of the childRoutes.